scripts/cluster/create_multinode_cluster.go

```diff
@@ -71,43 +73,22 @@ func CreateMultinodeCluster(stockContainerd string) error {
 	return nil
 }
 
-// Create kubelet service on master node
-func CreateMasterKubeletService() error {
-	utils.WaitPrintf("Creating kubelet service")
-	// Create service directory if not exist
-	_, err := utils.ExecShellCmd("sudo mkdir -p /etc/sysconfig")
-	if !utils.CheckErrorWithMsg(err, "Failed to create kubelet service!\n") {
-		return err
-	}
-	bashCmd := `sudo sh -c 'cat <<EOF > /etc/sysconfig/kubelet
-KUBELET_EXTRA_ARGS="--container-runtime=remote --v=%d --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
-EOF'`
-	_, err = utils.ExecShellCmd(bashCmd, configs.System.LogVerbosity)
-	if !utils.CheckErrorWithMsg(err, "Failed to create kubelet service!\n") {
-		return err
-	}
-	_, err = utils.ExecShellCmd("sudo systemctl daemon-reload")
-	if !utils.CheckErrorWithTagAndMsg(err, "Failed to create kubelet service!\n") {
-		return err
-	}
-
-	return nil
-}
-
 // Deploy Kubernetes
-func DeployKubernetes() error {
-
+func DeployKubernetes(haReplicaCount int) error {
 	utils.WaitPrintf("Deploying Kubernetes(version %s)", configs.Kube.K8sVersion)
-	masterNodeIp, iperr := utils.ExecShellCmd(`ip route | awk '{print $(NF)}' | awk '/^10\..*/'`)
-	if iperr != nil {
-		return iperr
-	}
-	shellCmd := fmt.Sprintf(`sudo kubeadm init --v=%d \
---apiserver-advertise-address=%s \
```

Comment:
We are loosing it in the default case, does it do it correctly by default?

Reply:
If you mean the api-server-advertise address, the cluster can work without that. I have tested it both with 1 CP and 3 CPs.